addons/backgrounds/README.md

```diff
@@ -37,35 +37,56 @@ Then write your stories like this:
 
 ```js
 import React from 'react';
-import { storiesOf } from "@storybook/react";
-import backgrounds from "@storybook/addon-backgrounds";
-
-storiesOf("Button", module)
-  .addDecorator(backgrounds([
-    { name: "twitter", value: "#00aced", default: true },
-    { name: "facebook", value: "#3b5998" },
-  ]))
-  .add("with text", () => <button>Click me</button>);
+import { storiesOf } from '@storybook/react';
+import { withBackgrounds } from '@storybook/addon-backgrounds';
+
+storiesOf('Button', module)
+  .addDecorator(
+    withBackgrounds([
+      { name: 'twitter', value: '#00aced', default: true },
+      { name: 'facebook', value: '#3b5998' },
+    ])
+  )
+  .add('with text', () => <button>Click me</button>);
 ```
 
-Of course it's easy to create a library module so you can re-use:
+You can add the backgrounds to all stories with `addDecorator` in `.storybook/config.js`:
 
 ```js
-import addonBackgrounds from "@storybook/addon-backgrounds";
-
-export const backgrounds = addonBackgrounds([
-  { name: "twitter", value: "#00aced", default: true },
-  { name: "facebook", value: "#3b5998" },
-]);
+import { addDecorator } from '@storybook/react'; // <- or your storybook framework
+import { withBackgrounds } from '@storybook/addon-backgrounds';
+
+addDecorator(
+  withBackgrounds([
+    { name: 'twitter', value: '#00aced', default: true },
+    { name: 'facebook', value: '#3b5998' },
+  ])
+);
 ```
 
+If you want to override backgrounds for a single story or group of stories, pass the `backgrounds` parameter:
+
 ```js
 import React from 'react';
-import { storiesOf } from "@storybook/react";
+import { storiesOf } from '@storybook/react';
+
+storiesOf('Button', module)
+  .addParameters({
+    backgrounds: [
+      { name: 'red', value: '#F44336' },
+      { name: 'blue', value: '#2196F3', default: true },
+    ],
+  })
+  .add('with text', () => <button>Click me</button>);
+```
 
-import { backgrounds } from "./my-lib";
+If you don't want to use backgrounds for a story, you can set the `backgrounds` parameter to `[]`, or use `{ skip: true }` to skip the addon:
```

Comment:
Not sure we really need to mention setting it to `[]` here?

Reply:
It's shorter, so it could see some use.

Reply:
I mean I not sure what makes sense if not `[]` meaning no backgrounds, right?

Reply:
In the case of the backgrounds addon the tab is always displayed--if you don't set any backgrounds it displays instructions about how to set it up.

If I understand what you are saying @Keraito we should do *three* things:

1. If no options/parameters are given / the addon is not turned on, display the instructions
2. If `[]` is passed, display the panel but with no options, perhaps a messaging saying no backgrounds were passed.
3. If `false` / `{ disabled: true }` is passed, remove the panel.

I think this is a good idea. Not sure if maybe it should be a separate PR though.